docker/Dockerfile.contrib

```diff
@@ -0,0 +1,79 @@
+FROM node AS build-zwavejs
+
+# RUN apk --no-cache add \
+#     git
+    # coreutils \
+    # linux-headers \
+    # alpine-sdk \
+    # python \
+    # openssl \
+
+WORKDIR /root/node-zwave-js
+COPY node-zwave-js .
+
+RUN npm install
+RUN npx lerna init
+RUN npx lerna bootstrap
+RUN npx lerna run build
+RUN npx lerna exec -- npm link
+RUN npm prune --production
```

Comment:
@AlCalzone I think this could be improved, also I think that we should add `npm install -g npm@7` before, right?

Reply:
Uh that seems to be based on the old description. The new one is a bit different:
https://zwave-js.github.io/node-zwave-js/#/development/installing-from-github

Since `npm workspaces`, it is no longer necessary to init or bootstrap lerna.

Reply:
I will try to remove the lerna steps.